tests/unittests/Foundation/NSJSONSerializationTests.m

```diff
@@ -124,13 +124,13 @@ void VerifyDataWithJSONObjectThrows(id testObject) {
     id testObject6 = @{ @"foo" : @"1", @"bar" : @"2" };
     id testObject7 = @{ @"foo" : @{ @"bar" : @"1" }, @"foo2" : @{ @"bar2" : @"2" } };
     id testObject8 = [NSUUID UUID];
-    id testObject9 = @{ @"key" : @"value", @"nullkey" : [NSNull null], @"array" : @[ @{ @"subnullkey" : [NSNull null] } ] };
```

Comment:
That is a fine idea. I think we have sufficient coverage as is, so we could punt it until we need to revamp this.

---
In reply to: [80984188](https://github.com/Microsoft/WinObjC/pull/1043#discussion_r80984188) [](ancestors = 80984188,80983725,80978644,80960041,80824620)